### PR TITLE
Added new OpenShift Virtualization inventory source to docs.

### PR DESCRIPTION
**Pull request by tvo318**
##### SUMMARY
Adds new inventory source docs for OpenShit Virtualization. Addresses issue #15274.

See rendered version here: https://ansible--15299.org.readthedocs.build/projects/awx/en/15299/userguide/inventories.html#openshift-virtualization

##### ISSUE TYPE
 - New or Enhanced Feature

##### COMPONENT NAME
 - Docs

##### AWX VERSION
latest

### CODE REVIEW

File: docs/docsite/rst/userguide/inventories.rst [hunk 1]
```diff
@@ -481,7 +481,7 @@ Inventory updates use dynamically-generated YAML files which are parsed by their
 - :ref:`ug_source_openstack`
 - :ref:`ug_source_rhv`
 - :ref:`ug_source_rhaap`
-- :ref:`ug_source_terraform`
```
Comment (TheRealHaoLiu) [inline docs/docsite/rst/userguide/inventories.rst:484]: why was this removed?

File: docs/docsite/rst/userguide/inventories.rst [hunk 2]
```diff
@@ -1120,6 +1120,46 @@ This inventory source uses the `terraform_state <https://github.com/ansible-coll
...
+  ::
+
+    ---
+    connections:
+    - namespaces:
+      - hao-test
+
+
+  .. image:: ../common/images/inventories-create-source-ocpvirt-example.png
+
```
Comment (TheRealHaoLiu) [inline docs/docsite/rst/userguide/inventories.rst:1155]: lets reference https://github.com/ansible-collections/kubernetes.core/blob/main/docs/kubernetes.core.k8s_inventory.rst#parameters

Comment (TheRealHaoLiu) [reply docs/docsite/rst/userguide/inventories.rst:1155]: similiar to what gcp inventory plugin 
```
Use the Source Variables field to override variables used by the gcp_compute inventory plugin. Enter variables using either JSON or YAML syntax. Use the radio button to toggle between the two. For a detailed description of these variables, view the [gcp_compute inventory plugin documentation](https://cloud.redhat.com/ansible/automation-hub/repo/published/google/cloud/content/inventory/gcp_compute).
```

File: docs/docsite/rst/userguide/inventories.rst [hunk 3]
```diff
@@ -1120,6 +1120,46 @@ This inventory source uses the `terraform_state <https://github.com/ansible-coll
...
+5. Save the configuration and click the **Sync** button to sync the inventory.
+
+As the sync runs, the results display in the job run output window. To access the job output while it is running, you can click on the spinning "Running" icon next to the **Last Job Status** field.
```
Comment (fosterseth) [inline docs/docsite/rst/userguide/inventories.rst]: I don't think this paragraph and the image below is necessary to include. This statement is not specific to this credential and would be the case for all the other inventory updates too.

File: docs/docsite/rst/userguide/inventories.rst (hunk 1 above)
Comment (tvo318) [reply docs/docsite/rst/userguide/inventories.rst:484]: Good catch, I must have accidentally typed over it instead of copying an entry for the new one.

File: docs/docsite/rst/userguide/inventories.rst (hunk 2 above)
Comment (tvo318) [reply docs/docsite/rst/userguide/inventories.rst:1155]: So for this, it needs to say `Use the Source Variables field to override variables used by the gcp_compute inventory plugin`?

Comment (tvo318) [reply docs/docsite/rst/userguide/inventories.rst:1155]: Or does it need to say `Use the Source Variables field to override variables used by the kubernetes.core.k8s inventory plugin`?

File: docs/docsite/rst/userguide/inventories.rst [hunk 4]
```diff
@@ -1120,6 +1120,46 @@ This inventory source uses the `terraform_state <https://github.com/ansible-coll
...
+
+1. To configure this type of sourced inventory, select **OpenShift Virtualization** from the Source field.
+2. The Create new source window expands with the required **Credential** field. Choose from an existing Kubernetes API Bearer Token credential. For more information, refer to :ref:`ug_credentials_ocp_k8s`. In this example, the ``cmv2.engineering.redhat.com`` credential is used.
+
+3. You can optionally specify the verbosity, host filter, enabled variable/value, and update options as described in the main procedure for :ref:`adding a source <ug_add_inv_common_fields>`. 
+
+4. Use the **Source Variables** field to override variables used by the ``controller`` inventory plugin. Enter variables using either JSON or YAML syntax. Use the radio button to toggle between the two. For more information on these variables, see the `Kubernetes (K8s) inventory source <https://github.com/ansible-collections/kubernetes.core/blob/main/docs/kubernetes.core.k8s_inventory.rst>`_ documentation for detail.
```
Comment (TheRealHaoLiu) [inline docs/docsite/rst/userguide/inventories.rst]: https://github.com/ansible-collections/community.kubevirt
here's the repo for the correct collection containing the inventory plugin

Comment (tvo318) [reply docs/docsite/rst/userguide/inventories.rst]: Using https://kubevirt.io/kubevirt.core/main/plugins/kubevirt.html#parameters

**Review by fosterseth — APPROVED** [review]
